autoread.js

```diff
@@ -106,14 +106,21 @@ function addSpeakBox(){
 	sbPlay.focus();
 }
 
-function handleCheckbox(){
-	let inputElems = currentLine.getElementsByTagName("input")
-	if(inputElems.length > 0 && inputElems[0].type == "checkbox"){
-		inputElems[0].click();
+function handleCheckbox(currentLine){
+	let inputElems = currentLine.getElementsByTagName("input");
+	if(inputElems.length > 0) {
```

Comment:
You can get rid of this outer check. If the inputElems.length == 0, then the for loop will just be skipped as i is already not less than 0.

Reply:
Done